source/index.md

```diff
@@ -2,6 +2,7 @@
 layout: index
 description: Hexo is a fast, simple & powerful blog framework powered by Node.js.
 subtitle: A fast, simple & powerful blog framework
+og_image: /themes/screenshots/landscape.png
```

Comment:
It would be better to set the official logo as the `og:image` for all pages. There's no need to change it page by page. So, we no need to specify `og_image` in each posts or pages.

Reply:
fixed with https://github.com/hexojs/site/pull/2154/commits/89563602f4b312e2511c2934dfa17d25755ce151